.travis.yml

```diff
@@ -2,7 +2,7 @@ os:
   - linux
 language: go
 go:
-  - 1.6
+  - 1.7
```

Comment:
Actually, hmm, maybe it makes sense to test against the oldest version we support though. Which case is more likely, that we inadvertently depend on super new standard library features, or that we inadvertently depend on deprecated features? I guess the obvious solution would be to test two versions on opposite ends of the spectrum?